glom/core.py

```diff
@@ -216,7 +228,7 @@ def format_oneline_trace(scope):
     # if the target is the same, don't repeat it
     segments = []
     prev_target = _MISSING
-    for scope, spec, target in _unpack_stack(scope):
+    for scope, spec, target, branches in _unpack_stack(scope):
```

Comment:
so based on the PR description, there's at most one branch here?

Reply:
it will need to do a recursive call with depth=depth+1 on each  branch

Reply:
hmm, I guess I might be asking a simpler question: `branches` can be an empty list or a list with one item?

Reply:
empty list

Reply:
```python
branches = scope.maps[0][CHILD_ERRORS][:-1]
```